model/channel.go

```diff
@@ -120,12 +120,18 @@ type ChannelModeratedRolesPatch struct {
 // PerPage number of results per page, if paginated.
 //
 type ChannelSearchOpts struct {
-	NotAssociatedToGroup   string
-	ExcludeDefaultChannels bool
-	IncludeDeleted         bool
-	ExcludeChannelNames    []string
-	Page                   *int
-	PerPage                *int
+	NotAssociatedToGroup    string
+	ExcludeDefaultChannels  bool
+	IncludeDeleted          bool
+	Deleted                 bool
```

Comment:
What is the difference between these? Is it possible to use just one?

Reply:
One of them is to include deleted results in the response and one of them is to return exclusively deleted results.
I dont think there is a way to do this without using 2 different options

Reply:
Ah I see. Perhaps change `Deleted` to `JustDeleted/OnlyDeleted`? 0/5. IMO Deleted and IncludeDeleted is a bit confusing. Deleted does not clarify that it exclusively returns deleted results.

Reply:
I think it makes more sense to leave it as `Deleted` instead of `OnlyDeleted` since `Private` and `Public` work the same way i.e. passing just `Private` returns "Only" Private channels